packages/jest-jasmine2/src/index.js

```diff
@@ -70,6 +71,40 @@ async function jasmine2(
   environment.global.describe.skip = environment.global.xdescribe;
   environment.global.describe.only = environment.global.fdescribe;
 
+  const bindEach = key => {
+    environment.global[key].each = (...args) => (title, test) =>
+      each
+        .withGlobal(environment.global)(...args)
+        [key](title, test);
+  };
+
+  const bindEachToNested = (parent, child) => {
+    environment.global[parent][child].each = (...args) => (title, test) =>
+      each
+        .withGlobal(environment.global)(...args)
+        [parent][child](title, test);
+  };
+
+  [
```

Comment:
There is only a finite set of these. It's more to type out, but can we be less smart and just manually inline this, like:

```
environment.global.test.each = …
environment.global.xtest.each = …
```

etc.?

If you look at line 67
https://github.com/facebook/jest/pull/6102/files#diff-e4399d1682ddb42f8a43f1a917d42d87R67

You'll also see that a lot of these are just aliases for each other, so you don't need to assign `.each` to all of them, just about half of them! :)

Reply:
Good shout, I've just updated :)